cmd/zed/zed.d/all-debug.sh

```diff
@@ -5,7 +5,9 @@
 # This can be a useful aid when developing/debugging ZEDLETs since it shows the
 # environment variables defined for each zevent.
 
+#shellcheck source=/dev/null
```

Comment:
`warning: Can't follow non-constant source. Use a directive to specify location. [SC1090]` is the error that is produced without them.